.code-samples.meilisearch.yaml

```diff
@@ -1065,3 +1065,19 @@ filtering_update_settings_1: |-
       "genres",
       "director"
     ]'
+get_documents_post_1: |-
+  curl \
+    -X POST http://localhost:7700/indexes/books/documents/fetch \
+    -H 'Content-Type: application/json' \
+    --data-binary '{ 
+      "filter": "(rating > 3 AND (genres = Adventure OR genres = Fiction)) AND language = English",
```

Comment:
In the known facet values of the movies dataset, `Fiction` does not exist. Did you mean `Science Fiction` ?

Reply:
I should have used the same index for one page. I'll keep that in mind next time 🙏